imap_processing/swe/l1b/swe_l1b_science.py

```diff
@@ -254,7 +257,8 @@ def populate_full_cycle_data(
             for step in range(180):
                 # Get esa voltage value from esa lookup table and
                 # use that to get row index in full data array
-                esa_voltage_value = esa_lookup_table.loc[esa_step_number]["esa_v"]
+                esa_voltage_value = esa_lookup_table.loc[esa_step_number]["esa_v"]  # type: ignore[attr-defined]
+                # TODO Change,  "list[Any]" has no attribute "loc"
```

Comment:
```suggestion
                esa_voltage_value = esa_lookup_table.loc[esa_step_number]["esa_v"]
```

I think this error will go away with the above fix (switching function to return type `Any`)?